test/packetimpact/tests/syn_with_same_ip_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The gVisor Authors.
```

Comment:
This test needs to be added to the BUILD file.

Also, you might need to sign the CLA.  Please check that you are passing the presubmit tests.

Reply:
a. [GES] You are suggesting to add “syn_with_same_ip_test.go” in BUILD file correct ?

b. [GES] We are already having github account with name “globaledgesoft-ltd” and CLA is already signed with this account. Going forward, will use same github account to submit code review

c. [GES] We need to run “gvisor/scripts/packetimpact_tests.sh” after adding new test code to gvisor/test/packetimpact/tests. And this will be presubmit tests and do test for netstack and gvisor. Is this correct ? Please confirm

Reply:
a. Yes.

b. Thanks.

c. Yes.  You can run `bazel test --cache_test_results=no --test_output=streamed test/packetimpact/tests:fin_wait2_timeout_linux_test` for example and it should work for you.  After you add the test to the BUILD file, then you need to replace `fin_wait2_timeout_linux_test` with your test name, like it might be `syn_with_same_ip_linux_test` and `syn_with_same_ip_netstack_test`.